hole/play.go

```diff
@@ -329,6 +329,12 @@ func play(
 			run.Stderr = `Quine in Go must not use "embed".`
 			return nil
 		}
+	case "iogii":
+		// Prevent trivial quines. Error out and return early.
+		if hole.ID == "quine" && len(code) > 0 && !strings.Contains(code, `"`) {
```

Comment:
Not sure if consistent with rules for other langs, but I think just disallowing quines that are just a number would be sufficient as there could be some interesting quines that don't use strings.

Reply:
Addressed with commit [e206374](https://github.com/code-golf/code-golf/pull/1765/commits/e2063744a54fb6ecac9b6836eb3075249937a171). It was just the first thing that seemed good enough to handle the job, and lack of sufficient knowledge and/or experience with the language is all that's to blame.